homeassistant/components/accuweather/weather.py

```diff
@@ -86,52 +100,60 @@ def condition(self):
             return None
 
     @property
-    def temperature(self):
+    def temperature(self) -> StateType:
         """Return the temperature."""
-        return self.coordinator.data["Temperature"][self._unit_system]["Value"]
+        return cast(
+            float, self.coordinator.data["Temperature"][self._unit_system]["Value"]
+        )
 
     @property
-    def temperature_unit(self):
+    def temperature_unit(self) -> str:
         """Return the unit of measurement."""
         return TEMP_CELSIUS if self.coordinator.is_metric else TEMP_FAHRENHEIT
 
     @property
-    def pressure(self):
+    def pressure(self) -> StateType:
         """Return the pressure."""
-        return self.coordinator.data["Pressure"][self._unit_system]["Value"]
+        return cast(
+            float, self.coordinator.data["Pressure"][self._unit_system]["Value"]
+        )
 
     @property
-    def humidity(self):
+    def humidity(self) -> StateType:
```

Comment:
There're more places, but you got the point 🙂
```suggestion
    def humidity(self) -> int:
```